kotlin/workflow-rx2/src/test/java/com/squareup/workflow/rx2/SubscriptionsTest.kt

```diff
@@ -169,8 +169,6 @@ class SubscriptionsTest {
 
   @Test fun `observable reports emissions`() {
     workflow.testFromStart(true) { host ->
-      // Get the next rendering to unblock the pipeline so the subscription can actually occur.
-      host.awaitNextRendering()
```

Comment:
These extra calls were all added to deal with the additional laziness. Using the correct dispatcher again means we can revert them.